gallery/src/components/NxTabs/NxTabsModalNoHeaderExample.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel,
+  NxModal
+} from '@sonatype/react-shared-components';
+
+const NxTabsModalNoHeaderExample = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [activeTabId, setActiveTabId] = useState(0);
+
+  function handleTabSelect(index: number) {
+    setActiveTabId(index);
+  }
+
+  return (
+    <>
+      <button onClick={() => setShowModal(true)} className="nx-btn">Open Modal</button>
+      { showModal &&
+        <NxModal onClose={modalCloseHandler} className="nx-modal--wide">
+          <div className="nx-modal-content">
+            <NxTabs activeTab={activeTabId} onTabSelect={handleTabSelect}>
+              <NxTabList aria-label="Tabs in a modal with no header">
+                <NxTab>Tab</NxTab>
+                <NxTab>Tab with longer name</NxTab>
+                <NxTab>Another Tab 3</NxTab>
+                <NxTab>Forth Tab</NxTab>
```

Comment:
Still misspelled in most examples

Reply:
> Still misspelled in most examples

https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/151565b015812ebb22b5fb2fcc53ec98daec903b